README.md

```diff
@@ -94,6 +94,14 @@ cordova.plugins.firebase.messaging.getToken("apns-string").then(function(token)
 });
 ```
 
+### clearMessages
```

Comment:
clearNotifications - is a better name, because notifications can be created locally too, not only from a FCM server

Reply:
Agree. Please respond on the other issue, I will commit both changes after that.